src/Ml.Cli.WebApp/ClientApp/src/Server/shared/ConfirmModal/ConfirmModal.js

```diff
@@ -0,0 +1,19 @@
+﻿import React from "react";
+import BooleanModal from "@axa-fr/react-toolkit-modal-boolean";
+import './ConfirmModal.scss';
+
+const ConfirmModal = ({title, isOpen, onCancel, onSubmit, children}) => (
```

Comment:
**suggestion**: typical component that need a storybook

Reply:
Done :)